content/integrations/aws.md

```diff
@@ -369,10 +369,7 @@ CloudWatch's api returns only metrics with datapoints, so if for instance an ELB
 
 
 When the Cross-Zone Load Balancing option is enabled on an ELB, all the instances attached to this ELB are considered part of all A-Zs (on CloudWatch’s side), so if you have 2 instances in 1a and 3 in ab, the metric will display 5 instances per A-Z.
```

Comment:
Should this be AZ or Availability Zone? Ive never seen it written as A-Z

Reply:
+1 to Ilan's comment.

Reply:
:+1: